week-2/debug/recipe.js

```diff
@@ -10,6 +10,13 @@ const recipe = {
   ingredients: ["olive oil", "tomatoes", "salt", "pepper"],
 };
 
+// console.log(`${recipe.title} serves ${recipe.serves}
+//   ingredients:
+// ${recipe}`);
+// this does not work because recipe does not include .ingredients and to show ingredients in a new line you need a for loop 
```

Comment:
the explanation here is not quite right. This does not work because ingredients's value if an array and to display an element from an arary you either access the using any of the array methods including loops or using index notations.

Reply:
Thank you for this explanation. I guess this is what I meant but I couldn't explain it.